src/components/ExportSection.vue

```diff
@@ -81,6 +81,7 @@
 					</template>
 					{{ t('user_migration', 'Export') }}
 				</Button>
+				<span v-if="estimatedSizeWithUnits" class="settings-hint">{{ t('user_migration', 'Estimated size: {estimatedSizeWithUnits}', { estimatedSizeWithUnits: this.estimatedSizeWithUnits }) }}</span>
```

Comment:
Github workflows are complaining about the usage of `this` here, i do not know enough about vue to understand what’s going on but it seems this is not used in the other blocks around, so maybe this is indeed a mistake?

Reply:
The lint warning will be fixed on main and backported https://github.com/nextcloud/user_migration/pull/182#issuecomment-1168982305